boot/bootutil/src/loader.c

```diff
@@ -2270,11 +2270,11 @@ context_boot_go(struct boot_loader_state *state, struct boot_rsp *rsp)
 #ifdef MCUBOOT_VALIDATE_PRIMARY_SLOT
 #ifdef PM_S1_ADDRESS
 	/* Patch needed for NCS. Image 1 primary is the currently
-	 * executing MCUBoot image, and is therefore already trusted and
-	 * does not need validation.
+	 * executing MCUBoot image, and is therefore already validated by NSIB and
+	 * does not need to also be validated by MCUBoot.
 	 */
-    bool image_needs_validation = BOOT_CURR_IMG(state) != 1;
-	if (image_needs_validation)
+    bool image_validated_by_nsib = BOOT_CURR_IMG(state) != 1;
```

Comment:
This does not have the right meaning AFAICT.

```suggestion
    bool image_not_validated = BOOT_CURR_IMG(state) != 1;
```

Reply:
I think that image_to_validate or something similar is better, having negatives on these variables can be confusing.